appinfo/Migrations/Version20191109111104.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace OCA\brute_force_protection\Migrations;
```

Comment:
Any reason why `brute_force_protection` is not in CamelCase?

Reply:
`MigrationService` cannot find migration classes that has '_' character in app name, it may indicate a bug on core. Because of that, I have been followed same syntax with previous migration class of this app.